### PR TITLE
Initialising subgraphs indexing Proxy Contracts

### PR DESCRIPTION
**Pull request by itsjerryokolo**

### CODE REVIEW

File: website/pages/en/deploying/hosted-service.mdx [hunk 1]
```diff
@@ -2,11 +2,11 @@
...
 
-This section will walk you through deploying a subgraph to the [hosted service](https://thegraph.com/hosted-service/).
+This section will walk you through deploying a subgraph to the [Hosted Service](https://thegraph.com/hosted-service/).
 
-If you don't have an account on the hosted service, you can sign up with your GitHub account. Once you authenticate, you can start creating subgraphs through the UI and deploying them from your terminal. The hosted service supports a number of networks, such as Polygon, Gnosis Chain, BNB Chain, Optimism, Arbitrum, and more.
+If you don't have an account on the hosted service, you can sign up with your GitHub account. Once you authenticate, you can start creating subgraphs through the UI and deploying them from your terminal. The Hosted Service supports a number of networks, such as Polygon, Gnosis Chain, BNB Chain, Optimism, Arbitrum, and more.
```
Comment (benface) [inline website/pages/en/deploying/hosted-service.mdx:9]: These changes look like regressions to me?

Comment (itsjerryokolo) [reply website/pages/en/deploying/hosted-service.mdx]: Yes, I fixed that now from the current version.

File: website/pages/en/deploying/hosted-service.mdx [hunk 2]
```diff
@@ -46,6 +46,19 @@ graph init --from-example --product hosted-service <GITHUB_USER>/<SUBGRAPH_NAME>
...
+You can use the command below.
+
+```sh
+graph init \
+  --product hosted-service
+  --from-contract <IMPLEMENTATION_ADDRESS> \
+  <GITHUB_USER>/<SUBGRAPH_NAME> [<DIRECTORY>]
+```
+
+## Supported Networks on the Hosted Service
```
Comment (benface) [inline website/pages/en/deploying/hosted-service.mdx:62]:
```suggestion
## Supported Networks on the hosted service
```

File: website/pages/en/deploying/hosted-service.mdx [hunk 3]
```diff
@@ -46,6 +46,19 @@ graph init --from-example --product hosted-service <GITHUB_USER>/<SUBGRAPH_NAME>
 
 The example subgraph is based on the Gravity contract by Dani Grant that manages user avatars and emits `NewGravatar` or `UpdateGravatar` events whenever avatars are created or updated. The subgraph handles these events by writing `Gravatar` entities to the Graph Node store and ensuring these are updated according to the events. Continue on to the [subgraph manifest](/developing/creating-a-subgraph#the-subgraph-manifest) to better understand which events from your smart contracts to pay attention to, mappings, and more.
 
-## Supported Networks on the hosted service
+
+### From a Proxy Contract
+
+To build a subgraph tailored for monitoring a Proxy contract, initialize the subgraph by specifying the address of the implementation contract. Once the initialization process is concluded, the last step involves updating the network name in the subgraph.yaml file to the address of the Proxy contract.
+You can use the command below.
```
Comment (benface) [inline website/pages/en/deploying/hosted-service.mdx:53]: I think this should either be part of the previous line, or there should be a line between the two if it's meant to be a separate paragraph.

Comment (itsjerryokolo) [reply website/pages/en/deploying/hosted-service.mdx:53]: Okay, I'll fix that.

**Review by benface — APPROVED** [review]
LGTM 👍

**Pull request merged**